apps/main/src/app/components/CalendarView.tsx

```diff
@@ -0,0 +1,87 @@
+import { SetStateAction, useState } from 'react';
+import { Calendar, dateFnsLocalizer } from 'react-big-calendar';
+import { format, parse, startOfWeek, getDay } from 'date-fns';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
+import { sl } from 'date-fns/locale';
+
+const locales = { sl: sl };
+
+const localizer = dateFnsLocalizer({
+  format,
+  parse,
+  startOfWeek: () => startOfWeek(new Date(), { weekStartsOn: 1 }),
+  getDay,
+  locales,
+});
+
+const events = [
+  {
+    title: 'Rezervacija - Stranka 1',
+    start: new Date(2025, 1, 18, 10, 0),
+    end: new Date(2025, 1, 18, 11, 0),
+  },
+  {
+    title: 'Rezervacija - Stranka 2',
+    start: new Date(2025, 1, 19, 14, 0),
+    end: new Date(2025, 1, 19, 15, 0),
+  },
+];
+
+export default function CalendarView() {
+  const [view, setView] = useState('month');
```

Comment:
```suggestion
  const [view, setView] = useState<string>('month');
```
z tem se lahku res une type warninge